File: frontend/src/types/Player.ts
export type Player = {
  ID: number,
  name: string,
  playerNumber: number,
  email: string,
  isVerified: boolean
}

export const DEFAULT_PLAYER: Player = {
  ID: 0,
  name: '',
  playerNumber: 0,
  email: '',
  isVerified: false,
}

File: api/datatypes/player.py
from pydantic import BaseModel


class Player(BaseModel):
    ID: int
    name: str
    playerNumber: int
    email: str
    isVerified: bool


class PlayerCreate(BaseModel):
    name: str
    playerNumber: int
    email: str
    password: str


class PlayerLogin(BaseModel):
    email: str
    password: str